docker/ceph/rpm/Dockerfile

```diff
@@ -2,9 +2,9 @@ ARG CENTOS_VERSION=8
 FROM rhcsdashboard/ceph-base:centos${CENTOS_VERSION}
 ARG CENTOS_VERSION
 
-# Centos met EOL and the content of the CentOS 8 repos has been moved to vault.centos.org
+# Centos met EOL and the content of the CentOS 8 repos has been moved to vault.epel.cloud
 RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-Linux-*
-RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-Linux-*
+RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.epel.cloud|g' /etc/yum.repos.d/CentOS-Linux-*
```

Comment:
Is it the same exact string over and over? Would it make sense to put this on a base container (e.g.: rhcsdashboard/ceph-base:8)? Keep in mind that this is probably going to break centos:9, so perhaps here we need a custom base for centos:8?

Reply:
is centos 9 going to be available? and if you are referring to centos-stream, there is no official centos-stream docker image. and if and when the official image is available I'll need to these changes.

There is this unofficial stream image though: https://hub.docker.com/r/tgagor/centos-stream